knime_extension/src/nodes/spatialstatistics.py

```diff
@@ -1044,3 +1067,243 @@ def execute(self, exec_context: knext.ExecutionContext, input_1, input_2):
         plt.title("Local Getis-Ord G Scatterplot")
 
         return knext.Table.from_pandas(gdf), knext.view_matplotlib(f)
+
+
+# new added nodes: 2023-11-27
+
+# Root path for all node icons in this file
+__NODE_ICON_PATH_2 = "icons/icon/Geolab/"
```

Comment:
Please move the icons into the proper folder instead of using another icon path here